gui/app/simple_clock.h

```diff
@@ -0,0 +1,55 @@
+#ifndef CLOCK_H
```

Comment:
Our style of include guards is CLOCK_H_; please check the other headers

Reply:
Again it's the Qt vs Google-style difference. Those files are generated by Qt Creator automatically.